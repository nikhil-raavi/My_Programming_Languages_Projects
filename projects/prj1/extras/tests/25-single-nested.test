var personPosition : record
  name: record
    firstName: string;
    lastName: string;
  end;
  position: record
    x: number;
    y: number;
    z: number;
  end;
end;