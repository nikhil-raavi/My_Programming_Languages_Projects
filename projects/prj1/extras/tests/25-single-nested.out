[
   [
      "personPosition",
      [
         [
            "name",
            [
               [
                  "firstName",
                  "string"
               ],
               [
                  "lastName",
                  "string"
               ]
            ]
         ],
         [
            "position",
            [
               [
                  "x",
                  "number"
               ],
               [
                  "y",
                  "number"
               ],
               [
                  "z",
                  "number"
               ]
            ]
         ]
      ]
   ]
]
